sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -118,8 +120,27 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putBoolean(KEY_TRACKING, tracking) }
     }
 
+    internal fun getListenToServerTrackingOptions(context: Context): Boolean {
+        return getSharedPreferences(context).getBoolean(KEY_SERVER_TRACKING, false)
+    }
+
+    internal fun setListenToServerTrackingOptions(context: Context, tracking: Boolean) {
+        getSharedPreferences(context).edit { putBoolean(KEY_SERVER_TRACKING, tracking) }
+    }
+
+    internal fun revertToFallbackTrackingOptions(context: Context) {
+        val fallbackTrackingOptions = getFallbackTrackingOptions(context)
+        if (fallbackTrackingOptions != null) {
+            setTrackingOptions(context, fallbackTrackingOptions)
+        }
```

Comment:
This can be simplified. This way, the `getListenToServerTrackingOptions` will return `false` next, since the result will be `null`.

```suggestion
        setTrackingOptions(context, getFallbackTrackingOptions(context))
```

Reply:
I see. this would mean we need to push the `null` check to `setTrackingOptions`. 

what if we wanted to `null` out the tracking options, though?

Reply:
oh, I think there may be some confusion around the methods. this one is `getFallbackTrackingOptions`, whereas the `Boolean` flag you might be thinking of is `getListenToServerTrackingOptions`

Reply:
oh yeah - i misread the change.